### DISCUSSION TITLE
Xフリースペースマップ→〇空き領域マップ

### DISCUSSION THREAD

**Discussion by koizumistr** [pgsql-jp/jpug-doc#2456] [NGとする語]
#2448 の件です。

**Comment by koizumistr** [comment]
いきなりDiscussionに書いてみましたが、これで合っているのでしょうか。

**Reply by koizumistr** [reply]
Create issue from discussion  というのもできるみたいですね。

**Reply by noborus** [reply]
> いきなりDiscussionに書いてみましたが、これで合っているのでしょうか。

はい。良いと思います。
マージされたのでNGリストの追加もお願いします。

**Reply by koizumistr** [reply]
#2463 でやってみました。
（ファイルで管理しているのだから、いつもと同じようにできてしまうんですね。）
